app/helpers/hyrax/batch_select_helper.rb

```diff
@@ -0,0 +1,11 @@
+# View Helpers for Hydra Batch Edit functionality
+module Hyrax
+  module BatchSelectHelper
+    # Displays the button to select/deselect items for your batch.  Call this in the index
+    # partial that's rendered for each search result.
+    # @param [Hash] document the Hash (aka Solr hit) for one Solr document
+    def button_for_add_to_batch(document)
```

Comment:
You stopped calling this method when you made this change: https://github.com/samvera-labs/hyrax/pull/1042/commits/d66ffc150d10c1c253915195025c66c2845cc21b#diff-4e8fdc051d085b4be70f397bbccffb11L26 So please remove this module.